src/protocols/index.js

```diff
@@ -30,8 +30,25 @@
 /** @typedef {import('./lending-protocol.js').RepayOptions} RepayOptions */
 /** @typedef {import('./lending-protocol.js').RepayResult} RepayResult */
 
+/** @typedef {import('./fiat-protocol.js').BuyOptions} BuyOptions */
+/** @typedef {import('./fiat-protocol.js').BuyCommonOptions} BuyCommonOptions */
+/** @typedef {import('./fiat-protocol.js').BuyExactCryptoAmountOptions} BuyExactCryptoAmountOptions */
+/** @typedef {import('./fiat-protocol.js').BuyWithFiatAmountOptions} BuyWithFiatAmountOptions */
+/** @typedef {import('./fiat-protocol.js').SellOptions} SellOptions */
+/** @typedef {import('./fiat-protocol.js').SellCommonOptions} SellCommonOptions */
+/** @typedef {import('./fiat-protocol.js').SellExactCryptoAmountOptions} SellExactCryptoAmountOptions */
+/** @typedef {import('./fiat-protocol.js').SellForFiatAmountOptions} SellForFiatAmountOptions */
```

Comment:
You can export only 'SellOptions':
```suggestion
/** @typedef {import('./fiat-protocol.js').SellOptions} SellOptions */
```

Reply:
https://github.com/nampc1/wdk-protocol-fiat-moonpay-fork/blob/main/src/moonpay-protocol.js#L332

Yes, I thought the same at first, but for maximum flexibility, I think we can still export those types. Please see the above example from MoonPay, it allows for selling with crypto amount only, not fiat amount.

Reply:
Just saw that while i was reviewing the repo. It makes sense, we can keep exporting the partial types as well.